README.md

```diff
@@ -64,7 +64,35 @@ Then you will see something like this:
 To get ruff automatically running install the following Plugin:
 - [VSCode - Ruff Plugin](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff)
 
-### Continuous Delivery
+
+## Testing
+
+A test suite of a typical Python package consists of unit, integration, and documentation tests
+(i.e. the code snippets embedded in the documentation),
+and it is truly important to write the tests to ensure that the code works as intended.
+
+The package template provides a minimal setup and the tests can be executed by invoking the Pytest runner:
```

Comment:
Just removed some words and added different punctuation. No hard feelings about it.

Maybe add a link to some article that explains what unit, integration and documentation tests are. I imagine some people might not know the difference. :) 

```suggestion
A test suite of a typical Python package consists of unit, integration, and documentation tests
(i.e. the code snippets embedded in the documentation).
It is important to write tests to ensure that your code works as intended.

This template provides a minimal setup for testing. Tests can be executed by invoking Pytest:
```

Reply:
Looks good, thank you!